css/structure/jquery.mobile.core.css

```diff
@@ -279,6 +279,7 @@ div.ui-mobile-viewport {
 	padding: 0;
 }
 .ui-btn-inline {
+	max-width:100% ;
```

Comment:
I don't think this should be part of the inline rule instead the `ui-select` rule I don't think we should restrict the width of inline buttons in general.  @jaspermdegroot @sfrisk thoughts?